docs/guides/development/cli.md

```diff
@@ -157,6 +157,38 @@ npx coda execute path/to/pack.ts Items --dynamicUrl=https://example.com/api/tabl
 ```
 
 
+### Running metadata functions
+
+More complicated Packs may have a lot of code outside of the core `execute` function, used to generate metadata like parameter autocomplete values, dynamic schemas, etc. You can run those functions using the CLI as well, using the same `execute` command but passing in additional information in name of the formula or sync table.
+
+```sh
+# Parameter autocomplete
+npx coda execute <formula name>:autocomplete:<paramName> [query] [params JSON]
+npx coda execute <sync name>:autocomplete:<paramName> [query] [params JSON]
+
+# Dynamic sync tables
+npx coda execute <sync name>:listDynamicUrls [parentUrl]
+npx coda execute <sync name>:searchDynamicUrls [query]
+npx coda execute <sync name>:getName
+npx coda execute <sync name>:getDisplayUrl
+npx coda execute <sync name>:getSchema [unused] [params JSON]
+
+# Two-way sync
+npx coda execute MySync:update [params..] [sync update array json]
+
+# Authentication
+npx coda execute Auth:getConnectionName
```

Comment:
Nit, we also have `Auth:getConnectionName`

Reply:
Did you mean to type `getConnectionId`? If so, I omitted that since we currently hide that function from developers, since it isn't used anywhere:

https://github.com/coda/packs-sdk/blob/8d4254faafb034c62fba03837de82d85b5a2f754/types.ts#L237-L246

Reply:
Oh yep, and thanks for the reminder, makes sense.